lib/hushed/date_service.rb

```diff
@@ -0,0 +1,55 @@
+module Hushed
+  class DateService
+
+    def build_date(date_string)
+      month = date_string.slice(0..1).to_i
+      day = date_string.slice(2..3).to_i
+      year = ("20" + date_string.slice(4..5)).to_i
+
+      Date.new(year, month, day)
+    end
+
+    def most_recent_day(date_array)
+      quick_sort(date_array).first
+    end
+
+    def quick_sort(date_array)
```

Comment:
that is a MUCH better solution. where is the `max` method defined though? i couldn't find anything like that in the docs that worked with dates... must have just missed something